test/test.expected

```diff
@@ -704,6 +704,25 @@ path-types.cooltt:55.10-55.17 [Info]:
   Computed normal form of pairext as A B p q h i => [ fst h i , snd h i ]
 
 --------------------[prelude.cooltt]--------------------
+--------------------[record.cooltt]--------------------
+record.cooltt:11.6-11.11 [Info]:
+  basic
+  : type
+  = sig (foo : nat)  (bar : (_x : nat) → nat)  
```

Comment:
Some funny whitespace going on here, but I don't care too much for now anyway 😆

Reply:
Oh I know what this is 🤦‍♀️. Should be easy to fix